test/e2e_node/eviction_test.go

```diff
@@ -395,6 +395,26 @@ var _ = SIGDescribe("PriorityMemoryEvictionOrdering", framework.WithSlow(), fram
 			{
 				evictionPriority: 1,
 				pod:              getMemhogPod("high-priority-memory-hog-pod", "high-priority-memory-hog", v1.ResourceRequirements{}),
+				prePodCreationModificationFunc: func(pod *v1.Pod) {
```

Comment:
non-blocking: in a followup PR/next time we change this code, let's use an explicit context, to be used below in the `List` call